app/lib/utils/translations/translations_transaction.dart

```diff
@@ -69,6 +112,37 @@ class TranslationsDeTransaction implements TranslationsTransaction {
       'Du hast dich abgemeldet vom nächsten Cycle. Du kannst dich wider anmelden, wenn du es dir anders überlegst.';
   @override
   String get unregisterParticipantNotificationTitle => 'Registrierung storniert';
+  @override
+  String get alreadyEndorsedErrorBody => 'Dieses Konto wurde bereits für diesen Key-Signing Cycle endorsed.';
+  @override
+  String get alreadyEndorsedErrorTitle => 'Bereits endorsed';
+  @override
+  String get noValidClaimsErrorBody =>
+      'Keine gültigen Bestätigungen wurden eingereicht. Hast du andere Teilnehmer gescannt?';
+  @override
+  String get noValidClaimsErrorTitle => 'Keine gültigen Bestätigungen';
+  @override
+  String get votesNotDependableErrorBody =>
+      'Es wurde nur die Hälfte oder weniger der zugewiesenen Teilnehmer bestätigt, oder einige Teilnehmer haben ihre Bestätigungen noch nicht eingereicht. Das verhindert das sofortige Auszahlen des Einkommens und du musst 48 Stunden warten.';
+  @override
+  String get votesNotDependableErrorTitle => 'Unzureichende Bestätigungen';
+  @override
+  String get insufficientFundsErrorBody =>
+      'Du hast nicht genügend Geld auf diesem Konto. Schaue auf der Webseite deiner lokalen gemeinschaft, wie du welches bekommen kannst.';
```

Comment:
Typo in my own text :)
```suggestion
      'Du hast nicht genügend Geld auf diesem Konto. Schaue auf der Webseite deiner lokalen Gemeinschaft, wie du welches bekommen kannst.';
```